hack-agenda.md

```diff
@@ -0,0 +1,14 @@
+#Hack-a-thon Agenda
+
+##Thursday
+- What is victory, clarify the vision and the first milestone targets
```

Comment:
Will talk at least briefly about where victory fits into the landscape - there are a LOT of people trying to do this same thing, and missing the target.